src/componentsList.ts

```diff
@@ -113,6 +113,22 @@ export const menuItems: MenuItems = {
     },
   },
   Switch: {},
+  Table: {
+    children: {
+      TableContainer: {},
+      Table: {},
+      TableCaption: {},
+      Thead: {},
+      Tbody: {},
+      Tfoot: {},
+      Tr: {},
+      Th: {},
+      Td: {},
+    },
+  },
+  TableRow: {
```

Comment:
what component is this?

Reply:
To add another row to the table, added it as a separate Meta component.

Reply:
Couldn't find it in MetaComponents. Only TableMeta and TableRowMeta are there

Reply:
Its there - TableRowMeta maps to TableRow! Check builder.ts file to see how it is getting built